packages/react-examples/.storybook/preview.js

```diff
@@ -21,7 +20,6 @@ const storyOrder = [
   'Migrations/Flex/Overview',
 ];
 
-addDecorator(withInfo);
```

Comment:
`withInfo` was replaced by some addon ?

Reply:
As of now, i haven't added anything to replace it yet. Since `addon-info` is deprecated, it was not playing nice with the new storybook webpack5 builder so Elizabeth and I decided to remove it for now (see [here](https://github.com/microsoft/fluentui/pull/18693#discussion_r663141571))

Reply:
sounds good, vNext uses storybook Docs anyways with typescript doc generation

Reply:
with that we can also remove our manual typings 
![2021-07-20 at 19 24](https://user-images.githubusercontent.com/1223799/126368464-50f2ff0c-60c0-4da7-8714-9bc2c104d00f.png)

Reply:
I'll create a separate issue for this!